docs/User/SWCI_APIs.md

```diff
@@ -38,7 +38,7 @@ The python3 program needs to import the SWCI class, and then use the below APIs.
 |`listTasks()`|This method displays the list of tasks that are registered into the Swarm Learning network.| |
 |`listTrainingContracts()`|This method displays the list of training contracts registered with Swarm Learning network.| |
 |`ls()`|This method displays the directory contents of the SWCI container.|`optStr=''`|
-|`plotTopology()`|This method displays the PNG object showing the current topology of the Swarm Learning network.| |
+|`plotTopology()`|This method displays the PNG object showing the current topology of the Swarm Learning network.<br> By default, it will display the Swarm Node type and Host IP on which the node is running.<br> User can pass additional attributes which they want to see on the plot. These additional attributes are: 'Port', 'ContainerName', 'UUID', 'parentUUID' and 'i-am-alive'.<br> **NOTE**: Color codes are hexadecimal triplets representing the colors red, green, and blue (#RRGGBB).|SNColour="#ADD8E6",<br> SWOPColor="#33FF33",<br> SLColor="#FFCCCB", <br> attrs=['ContainerName']|
```

Comment:
Make it 2 para's.

new-line. 

"User can pass additional attributes which they want to..."

Reply:
This para can be moved to 3rd column - "User can pass additional attributes which they want to see on the plot... "